admin-dev/themes/new-theme/js/app/utils/ip-input.js

```diff
@@ -23,13 +22,19 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
-
-header('Expires: Mon, 26 Jul 1997 05:00:00 GMT');
-header('Last-Modified: '.gmdate('D, d M Y H:i:s').' GMT');
-
-header('Cache-Control: no-store, no-cache, must-revalidate');
-header('Cache-Control: post-check=0, pre-check=0', false);
-header('Pragma: no-cache');
-
-header('Location: ../../../../../../../');
-exit;
+global.IpInput = {};
+IpInput.addRemoteAddr = (target) => {
```

Comment:
as a last comment, some failures from codacy can be fixed for free => https://www.codacy.com/app/PrestaShop/PrestaShop/pullRequest?prid=1148080

Reply:
Indeed, I totally missed that solution by migrating the code.